frontend/public/components/_resource.scss

```diff
@@ -144,4 +144,12 @@
   &__resource-name {
     min-width: 0; // required so co-break-word works
   }
+  &__resource-status {
+    padding-left: 8px;
+    white-space: nowrap;
+
+    .pf-c-badge {
```

Comment:
I'd suggest passing in a specific `className` prop to `Badge` like `co-resource-item__resource-status-badge` instead of relying on nesting styles.